rsvps/exporter/Exporter.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+class Exporter
+{
+    public function init(): void
+    {
+        add_action('export_filters', [$this, 'renderExportFilters']);
+        add_action('export_wp', [$this, 'generateCSV']);
+    }
+
+    public function renderExportFilters(): void
+    {
+        wp_enqueue_script('jquery-ui-datepicker');
+        wp_enqueue_style('jquery-ui-datepicker-style',
+            '//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css');
```

Comment:
Do we actually need to enqueue this stylesheet? Doesn't WP have the datepicker styles built-in somewhere?

Reply:
I was surprised, too, but it's necessary. Without this the datepicker was missing styles and had no background. They're only enqueued on this one page so it shouldn't have any negative effects elsewhere.